pkg/agent/flags/flag_controller.go

```diff
@@ -101,29 +104,132 @@ func (fc *FlagController) notifyObservers(flagKeys ...keys.FlagKey) {
 	}
 }
 
+func (fc *FlagController) AutoloadedExtensions() []string {
+	return fc.cmdLineOpts.AutoloadedExtensions
+}
+
+func (fc *FlagController) SetKolideServerURL(url string) error {
+	return fc.setControlServerValue(keys.KolideServerURL, []byte(url))
+}
+func (fc *FlagController) KolideServerURL() string {
+	return NewStringFlagValue(
+		WithDefaultString(fc.cmdLineOpts.KolideServerURL),
+	).get(fc.getControlServerValue(keys.KolideServerURL))
+}
+
+func (fc *FlagController) SetKolideHosted(hosted bool) error {
+	return fc.setControlServerValue(keys.KolideHosted, boolToBytes(hosted))
+}
+func (fc *FlagController) KolideHosted() bool {
+	return NewBoolFlagValue(WithDefaultBool(false)).get(fc.getControlServerValue(keys.KolideHosted))
+}
```

Comment:
I don't think we're going to override this one. It's really just a thing to set other defaults with. (And we might phase it out)